### PR TITLE
Solidarity Update from @netanoir

### PR DESCRIPTION
**Pull request by shiffman**
I am bring over this pull request by @netanoir https://github.com/netanoir/ml5-website/pull/1

> * remove Tony McDade's deadname
> * add names of Black women missing from this list: Regis Korchinski-Paquet, Nina Pop, Monika Diamond
> * remove donation suggestions to funds that have been criticized by the community, especially #8CantWait
> * add Breonna Taylor's fund to donation suggestions
> * add donation suggestions to lesser known & vetted funds: local bail funds list, Emergency Release Fund, Survived & Punished, Movement for Black Lives
> * reorganize donation suggestions to prioritize local & lesser known funds

### CODE REVIEW

**Review by bomanimc — APPROVED** [review]
Catching up on the statement work that has been done. @netanoir these changes to the statement look good to me! Thanks @netanoir for noticing these issues and opening a PR with these important fixes!

**Comment by bomanimc** [comment]
@netanoir @shiffman: While we're on the topic of conversation, a few others I think we could add here:
- Sandra Bland
- David McAtee

File: src/templates/home-page.js [hunk 1]
```diff
@@ -73,13 +74,14 @@ const BLMSolidarityModal = () => {
...
         </main>
         <p style={{marginBottom:"1rem"}}>
-            Please consider donating to <a href="https://blacklivesmatter.com/" noreferrer>Black Lives Matter</a>, 
-            <a href="https://www.naacpldf.org/" noreferrer>The NAACP Legal Defense and Educational Fund</a>, 
-            <a href="https://eji.org/">The Equal Justice Initiative</a>, 
-            <a href="https://www.wetheprotesters.org/">We The Protesters and Campaign Zero</a>,
-            <a href="https://codecooperative.org/">Code Cooperative</a>,
-             and 
-            <a href="https://www.gofundme.com/f/georgefloyd"> George Floyd Memorial Fund</a>.
+            Please consider donating to <a href="https://bailfunds.github.io/">your local bail fund</a>,
```
Comment (bomanimc) [inline src/templates/home-page.js]: I noticed a tiny UI nit: We don't render spaces in between items on the list of links. If we want to maintain the multiline layout in the code (which definitely makes it easier to read), maybe we can add some non-breaking spaces with `&nbsp;` after each comma.

**Comment by shiffman** [comment]
@bomanimc I addressed your comments in https://github.com/ml5js/ml5-website/pull/159/commits/cabfc11a1f883f16510c89ea8836bc6c0fbe1395 and https://github.com/ml5js/ml5-website/pull/159/commits/d7f2914e0992917111b4c5b31177f27ed5b169ec. I had some errors trying to build the site locally to double-check but since all the tests passed do you think it's ok to merge?

**Comment by shiffman** [comment]
I'll also note that I made arbitrary decisions with the order of the names and I am glad to re-arrange based on any feedback of course.

**Comment by bomanimc** [comment]
@shiffman Hmmm, I'm not sure about the local tests (I also haven't worked in this repo for a while), but I think we should be okay to merge since the deploy preview looks good: https://deploy-preview-159--ml5-website.netlify.app/

Regarding the ordering, we could consider ordering reverse chronologically on the date that each person passed on, but I think having a less computable ordering makes sense since it gives us the opportunity to include names that have not been called on enough to the top of the list (that's why I like the placement of the names of Black women that @netanoir added). 

I think the order is probably okay overall. One change I'd suggest:
- Move Breanna Taylor's to be among the first four names on the list since Breanna's name is one of the most widely know at the moment (along with George Floyd and Ahmaud Arbery). If we're starting the list with these more well-known names, my preference would be to put Breanna Taylor's name in the first position on the list.

**Comment by shiffman** [comment]
I've adjusted the order according to @bomanimc's feedback and checked the deploy-preview. I am going to merge now. If there is additional feedback or comment I am glad to make further adjustments. Thank you everyone for your time working on this and thoughtfulness.

**Comment by joeyklee** [comment]
👏🚀

**Pull request merged**